module/VuFind/src/VuFind/ILS/Driver/MultiBackend.php

```diff
@@ -192,7 +192,17 @@ public function getStatuses($ids)
     {
         $items = [];
         foreach ($ids as $id) {
-            $items[] = $this->getStatus($id);
+            try {
+                $items[] = $this->getStatus($id);
+            } catch (ILSException $e) {
+                $items[] = [
+                    [
+                        'id' => $id,
+                        'error' => 'An error has occurred',
+                        'availability_message' => 'status_unknown_message'
```

Comment:
If you find a way to display the full unknown message above, it may be necessary to devise a different mechanism for filling it in in this situation. Maybe any entry with 'error' set automatically gets 'availability_message' set to unknown.

Reply:
That was more of a fallback but I think I'll remove it.